fortitude/src/options.rs

```diff
@@ -279,4 +280,32 @@ pub struct CheckOptions {
         "#
     )]
     pub per_file_ignores: Option<FxHashMap<String, Vec<RuleSelector>>>,
+
+    /// Options for the `exit-or-cycle-in-unlabelled-loops` rule
+    #[option_group]
+    pub exit_labelled_loops: Option<ExitLabelledLoopOptions>,
+}
+
+/// Options for the `exit-or-cycle-in-unlabelled-loops` rule
+#[derive(
+    Clone, Debug, PartialEq, Eq, Default, OptionsMetadata, CombineOptions, Serialize, Deserialize,
+)]
+#[serde(deny_unknown_fields, rename_all = "kebab-case")]
+pub struct ExitLabelledLoopOptions {
+    /// Whether to check for `exit`/`cycle` in unlabelled loops only if the loop has at
+    /// least one level of nesting
+    #[option(
+        default = "false",
```

Comment:
Generally speaking, should we default to the stricter interpretation of the rule and require the user to tell us to be nicer, or would the opposite be preferable? Personally, I think I would always want this to be `true`.

Reply:
Currently the default is for the stricter version, triggering on `exit`/`cycle` in just one level of unlabelled loop. Turning this on makes it need an extra level of nested loop.